tests/unit/test_utils.py

```diff
@@ -1,3 +1,4 @@
+# pylint:disable=protected-access,no-self-use
```

Comment:
@jbzdak (1) please don't do that and (2) why it is added here in the first place?